samples/bluetooth/peripheral_csc/prj.conf

```diff
@@ -2,5 +2,8 @@ CONFIG_ENTROPY_GENERATOR=y
 CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_BT=y
 CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_GATT_BAS=y
+CONFIG_BT_GATT_DIS=y
+CONFIG_BT_GATT_DIS_MANUF="ACME"
```

Comment:
Yea... good joke. Be careful with that as such company really exist, even more than one:
https://www.acme.eu/en-us
https://acme.com/
http://acme-world.com/about/

Reply:
In USB library we have: "USB manufacturer name". Looks just more professional.